.github/workflows/build-test

```diff
@@ -104,7 +104,12 @@ for MODULE in $MODULES_TO_TEST; do
                 dotnet iqsharp install --user
             fi
 
-            pytest --doctest-modules
+            # Disable the remote tests for pytket-aqt, -ionq, and -qsharp for now
+            if [[ "${MODULE}" = "pytket-aqt" || "${MODULE}" = "pytket-ionq" || "${MODULE}" = "pytket-qsharp" ]]; then
+                unset PYTKET_RUN_REMOTE_TESTS; pytest --doctest-modules
+            else
+                pytest --doctest-modules
+            fi
```

Comment:
The problem with doing `PYTKET_RUN_REMOTE_TESTS=0` is that in the python code, if you do `os.getenv("VAR")`, it will only return None if the variable does not exist. By setting it to 0, then the code for the remote tests will execute as the value is not None but 0.

Now, it's true that doing the `unset PYTKET_RUN_REMOTE_TESTS` will leave the var unset, but that's only for the rest of this bash script. It will not modify the value of the variable from the caller. Since we're now executing the script just once per module (as we're running them in parallel), I don't think this is too much problem.

Still, I could save the value of the variable and restore it later after running the pytest. Should I apply that change?

Reply:
I think we should either adjust the python code so that it treats the value 0 equivalently to None (unset), or modify this script so that it clearly only tests one module: read from a variable like `MODULE`, and get rid of the loops `for MODULE in $MODULES_TO_TEST`. Otherwise it's very confusing since the script is obviously written to work with a list of modules but doesn't.